File: frontend/src/types/productionLoad.ts
export interface IProductionLoad {
	id: string | null
	date?: string
	sector: string | null
	days: number | null
	quantity: number | null
	money: number | null
}

export interface IProductionLoadDTO {
	id: string
	date: number
	sector: string
	days: number
	quantity: number
	money: number
}

File: backend/internal/models/production_load.go
package models

type ProductionLoad struct {
	Id       string `json:"id" db:"id"`
	Date     int64  `json:"date" db:"date" binding:"required"`
	Sector   string `json:"sector" db:"sector" binding:"required"`
	Days     int    `json:"days" db:"days" binding:"required"`
	Quantity int    `json:"quantity" db:"quantity"`
}
